src/client/pages/Checkout/shared/Footer.tsx

```diff
@@ -47,6 +48,32 @@ const InnerWrapper = styled.div`
 `
 const ButtonWrapper = styled.div`
   justify-self: end;
+
+  ${Button} {
+    padding: 0.75rem 1rem;
+
+    ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+      padding: 0.75rem 2rem;
+    }
+  }
+
+  ${LinkButton} {
+    padding: 0.75rem 1rem;
+
+    ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+      padding: 0.75rem 2rem;
+    }
+  }
+
+  ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+    ${Button} {
+      padding: 0.75rem 2rem;
+    }
+
+    ${LinkButton} {
```

Comment:
And this is the same as `${LinkButton} ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY}` above?